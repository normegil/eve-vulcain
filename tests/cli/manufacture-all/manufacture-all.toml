bin.name = "eve-vulcain"
args = "manufacture all -vvv"
status.code = 0
