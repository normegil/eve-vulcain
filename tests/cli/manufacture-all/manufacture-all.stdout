                            Drone Link Augmentor I               1,331                                       -836,802.79 ISK/h
                                 Damage Control II              10,822                                       -944,809.67 ISK/h

